client/data/paid-newsletter/use-paid-newsletter-query.ts

```diff
@@ -52,7 +52,7 @@ export interface Product {
 }
 
 export interface Plan {
-	active_subscriptions: boolean;
+	active_subscriptions: number;
```

Comment:
@lezama do you know about the shape of this object and why it was set to `bool`, not `number`? Can it be both?

Reply:
I don't know about it 😅